resources/CMakeLists.txt

```diff
@@ -29,9 +29,10 @@ if (HAVE_OPENCL)
 endif()
 
 # Server, also install server subdirectory
+SET(SERVER_DEST_RESOURCE_FILES "")
```

Comment:
Can we use lowercase here?
In general this is preferred in newer cmake code (I plan to do a mass reformat on uppercase ones)

Reply:
```suggestion
set(SERVER_DEST_RESOURCE_FILES "")
```

Reply:
Sure! I was thinking of doing it to the macro too, but wanted to keep the diff small and understandable